dev/codeserver/java/com/google/gwt/dev/codeserver/dev_mode_on.js

```diff
@@ -115,14 +115,15 @@
     return dialog;
   }
 
-  function makeBookmarklet(name, javascript) {
+  function makeBookmarklet(name, javascript, javascriptFunction) {
```

Comment:
Instead of passing `javascript` as a separate argument causing duplication we could compute it
```
var javascript = '('+(javascriptFunction+"").replace(/\s+/g,' ')+')()';
```
(the replace is just to keep it shorter)

Reply:
@zbynek Thanks for this proposal. The missing part would be to replace the "placeholder" variables ("codeserver_url", "module_name" and "bookmarklets_js") later on with real (string) values. Not sure, if this would be more readable at the end.